packages/desktop-client/src/components/transactions/TransactionsTable.jsx

```diff
@@ -1349,7 +1354,14 @@ const Transaction = memo(function Transaction(props) {
   );
 });
 
-function TransactionError({ error, isDeposit, onAddSplit, onDistributeRemainder, style, canDistributeRemainder }) {
```

Comment:
All of these other changes in the file: automatically applied by prettier. Just code formatting changes.

I wonder why linter was not breaking about them.. weird